modules/editing-the-dashboard-configuration.adoc

```diff
@@ -0,0 +1,44 @@
+:_module-type: PROCEDURE
+
+[id="editing-the-dashboard-configuration_{context}"]
+= Editing the dashboard configuration
+
+[role='_abstract']
+As an administrator, you can customize the interface of the dashboard by editing the dashboard configuration.
+
+.Prerequisites
+* You have cluster administrator privileges for your {openshift-platform} cluster.
```

Comment:
Small nit, to modify our OdhDashboardConfig you only need to be a Product Admin now -- we leveled this up in 2.17

As an Admin you get all of these powers over the resource:
```
- apiGroups:
  - opendatahub.io
  resources:
  - odhdashboardconfigs
  verbs:
  - get
  - list
  - watch
  - create
  - update
  - patch
```
[Source](https://github.com/opendatahub-io/opendatahub-operator/blob/abe30504686980f9fa648468aabdd18460e306ac/internal/controller/services/auth/resources/admingroup-role.tmpl.yaml#L80C1-L90C10)